pkg/karmadactl/cmdinit/kubernetes/deployments.go

```diff
@@ -65,7 +65,6 @@ func (i *CommandInitOption) karmadaAPIServerContainerCommand() []string {
 		"--allow-privileged=true",
 		"--authorization-mode=Node,RBAC",
 		fmt.Sprintf("--client-ca-file=%s/%s.crt", karmadaCertsVolumeMountPath, options.CaCertAndKeyName),
-		"--enable-admission-plugins=NodeRestriction",
```

Comment:
I don't remember why we need the [NodeRestriction](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) plugin for `karmada-apiserver`. @mrlihanbo can you remind me?

@lonelyCZ Can you explain why you remove it?

Reply:
Mainly lease is restricted at `kube-node-lease` namespace.

```
I0822 07:30:51.558065       1 leaderelection.go:253] failed to acquire lease karmada-system/karmada-agent-member68
E0822 07:30:54.921883       1 leaderelection.go:367] Failed to update lock: leases.coordination.k8s.io "karmada-agent-member68" is forbidden: can only access leases in the "kube-node-lease" system namespace
```

Reply:
Because it can only use `system:node:` CN and `system:nodes` OG for agent cert,  to reuse the controller of automatically approving csr.